lib/rule-context.js

```diff
@@ -92,18 +92,18 @@ function mergeFixes(fixes, sourceCode) {
     const start = fixes[0].range[0];
     const end = fixes[fixes.length - 1].range[1];
     let text = "";
-    let lastPos = Math.max(0, start);
+    let lastPos = Number.MIN_SAFE_INTEGER;
```

Comment:
Yeah :smile:

I expect that integer variables are faster by compiler optimization.